### PR TITLE
Add notes about mobile data sensors

### PR DESCRIPTION
**Pull request by frimtec**
Documentation for home-assistant/android#1317

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @frimtec, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;companion.home-assistant#435).

Once you do that we will be able to review and accept this pull request.

Thanks!

**Comment by TomBrien** [comment]
Parent: home-assistant/android#1317

File: docs/core/sensors.md [hunk 1]
```diff
@@ -434,6 +435,14 @@ This sensor displays exactly what caused the last update of location and sensor
 ![Android](/assets/android.svg)<br />
 This sensor will reflect the current level of illuminance the device detects. The sensor updates during the normal sensor update interval or with the other sensor updates and makes use of [Environment Sensors](https://developer.android.com/guide/topics/sensors/sensors_environment).
 
+## Mobile Data Sensors
+![Android](/assets/android.svg)<br />
+Several different sensors around the state of mobile data.
```
Comment (dshokouhi) [inline docs/core/sensors.md:440]: Can you please include a link to the Android API that we use here?

**Review by dshokouhi — CHANGES_REQUESTED** [review]
Let's add a link to the Android API being used to match the rest of the Android sensors.

File: docs/core/sensors.md (hunk 1 above)
Comment (frimtec) [reply docs/core/sensors.md]: Done.

**Review by dshokouhi — APPROVED** [review]
Looks good to me, can be merged once the parent PR is merged

**Pull request merged**